Dockerfile

```diff
@@ -30,10 +30,11 @@ RUN cd /opt && tar -xvzf android-sdk.tgz
 RUN cd /opt && rm -f android-sdk.tgz
 
 # Grab what's needed in the SDK
+RUN echo y | android update sdk --no-ui --all --filter tools > /dev/null 
```

Comment:
They made some changes to the command line tool in the last versions, it might not even work anymore....